city_scrapers/spiders/chi_state_university.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiStateUniversitySpider(CityScrapersSpider):
+    name = "chi_state_university"
+    agency = "Chicago State University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.csu.edu/boardoftrustees/dates.htm"]
+
+    def parse(self, response):
+        for item in response.css(".table tr td li"):
+            extracted = item.css("::text").extract()
+            meeting = Meeting(
+                title=self._parse_title(response),
+                classification=self._parse_classification(),
+                start=self._parse_start(extracted),
```

Comment:
I made a note about checking the status below, but it may be a little more involved than that. Usually it looks like each bullet will include a date that a meeting has been rescheduled to. If there are two dates we should just use the second one, and it should not be marked cancelled

Reply:
So I don't have to grab the Rescheduled status?

Reply:
You'll still need to handle it, but by checking for dates in the string and not necessarily using it directly in `_get_status`